web/src/main/java/org/fao/geonet/monitor/link/LinkInfo.java

```diff
@@ -0,0 +1,91 @@
+package org.fao.geonet.monitor.link;
+
+import org.jdom.Element;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class LinkInfo {
+
+    private List<CheckInfo> checkInfoList;
+
+    private final Element onlineResource;
+
+    private final LinkCheckerInterface linkChecker;
+
+    public LinkInfo(Element onlineResource, LinkCheckerInterface linkChecker) {
+        checkInfoList = Collections.synchronizedList(new ArrayList<CheckInfo>());
```

Comment:
Why the need for a synchronized list?  In any case, we're not synchronizing on it when iterating (see http://docs.oracle.com/javase/7/docs/api/java/util/Collections.html#synchronizedList(java.util.List)) so in case we *do* actually need it, then we've possibly got a bug.